lib/puppet/http/resolver/serverlist.rb

```diff
@@ -0,0 +1,37 @@
+class Puppet::HTTP::Resolver::ServerList < Puppet::HTTP::Resolver
+  def initialize(client, server_list: Puppet[:server_list], default_port: Puppet[:masterport])
```

Comment:
Could you make `server_list` and `default_port` required keyword arguments like `(client, server_list:, default_port:)`